### PR TITLE
bpo-31822 Document that urllib.parse.{Defrag,Split,Parse}Result are namedtuples

### PR DESCRIPTION
**Pull request by lisroach**
https://bugs.python.org/issue31822

### CODE REVIEW

File: Doc/library/urllib.parse.rst [hunk 1]
```diff
@@ -88,8 +89,9 @@ or on combining URL components into a URL string.
    or query component, and :attr:`fragment` is set to the empty string in
    the return value.
 
-   The return value is actually an instance of a subclass of :class:`tuple`.  This
-   class has the following additional read-only convenience attributes:
+   The return value is actually an instance of a subclass of :class:`tuple` (see
+   :class:`~collections.namedtuple`).
```
Comment (merwok) [inline Doc/library/urllib.parse.rst]: I would link to the glossary here too: `(a :term:`named tuple` with 6 items)`

**Comment by lisroach** [comment]
I think 'skip-news' could be added here to get this passing checks.

**Comment by csabella** [comment]
I resolved the merge conflict.  @lisroach and @merwok, I believe this one can be merged, but you may want to look it over once more.  Thanks!

**Review by csabella — APPROVED** [review]
LGTM

File: Doc/library/urllib.parse.rst [hunk 2]
```diff
@@ -128,6 +130,24 @@ or on combining URL components into a URL string.
    normalization (as used by the IDNA encoding) into any of ``/``, ``?``,
    ``#``, ``@``, or ``:`` will raise a :exc:`ValueError`. If the URL is
    decomposed before parsing, no error will be raised.
+
```
Comment (tirkarthi) [inline Doc/library/urllib.parse.rst]: This line has a trailing space causing CI failure. Otherwise LGTM. Thanks.

**Review by merwok — APPROVED** [review]

**Comment by csabella** [comment]
Thanks @lisroach for the PR and @merwok and @tirkarthi for the reviews.

**Pull request merged**